### ISSUE TITLE
Can't install rpmbuilder

### ISSUE THREAD

**Issue by Extor** [simtechdev/smtk-repo#2]
``` bash
sudo yum install -y https://release.yum.smtk.us/smtk-release-7.noarch.rpm
sudo yum -y install rpmbuilder
...
---> Пакет perl-parent.noarch 1:0.225-244.el7 помечен для установки
---> Пакет tmux.x86_64 0:2.1-0.el7.centos помечен для установки
--> Обработка зависимостей: libevent-2.0.so.5()(64bit) пакета: tmux-2.1-0.el7.centos.x86_64
--> Проверка зависимостей окончена
Ошибка: Пакет: tmux-2.1-0.el7.centos.x86_64 (smtk-release)
            Необходимо: libevent-2.0.so.5()(64bit)
 Вы можете попробовать --skip-broken чтобы обойти проблему
 Вы можете попробовать запустить: rpm -Va --nofiles --nodigest

```

it works after install libevent-2.0.21

**Comment by gongled** [comment]
Thanks for your reply. It seems to be a bug.

**Comment by gongled** [comment]
Fixed (0651c483ff94509f0967e004a033be22a86395fe). I've already moved `tmux` package into the release branch.
